hack/cluster/cluster.go

```diff
@@ -11,6 +11,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/davecgh/go-spew/spew"
```

Comment:
I understand why it's needed for a library like this, but [this module's dependency on `unsafe`](https://github.com/davecgh/go-spew#unsafe-package-dependency) scares me a bit for it's maintainability over time.

Reply:
I'm open to alternatives.  I chose this library because it was already vendored in through an indirect dependency.  The [OData error response](https://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_ErrorResponse) structure can be a bit unwieldy and I just wanted something that can fully dump it to logs.

Reply:
ah, I meant to review if it was already there (was reviewing one commit at a time), seeing as it's already vendored, gonna close this. thanks!